pkg/cxl/cxl-util.go

```diff
@@ -279,13 +283,20 @@ func (c *CxlDev) GetPcieHdr() *PCIE_CONFIG_HDR {
 
 // return the Vendor Info of the PCIe/CXL device
 func (c *CxlDev) GetVendorInfo() string {
```

Comment:
For this function, one suggestion is to use an `init()` function to create a map data structured indexed by the 4 vendor id. That way, all subsequent calls to `GetVendorInfo()` would be very fast. This can be done in a follow on pull request.